lib/source.c

```diff
@@ -29,6 +29,7 @@
 #include <tlog/misc.h>
 #include <tlog/source.h>
 
+
```

Comment:
Nah, not an empty line here, I was talking about that space in front of `last_timestamp`.